src/devtools/client/themes/variables.css

```diff
@@ -87,6 +87,7 @@
   --grey-10-a20: rgba(249, 249, 250, 0.2);
   --grey-10-a25: rgba(249, 249, 250, 0.25);
   --grey-10-a30: rgba(249, 249, 250, 0.3);
+  --grey-10-a40: rgba(249, 249, 250, 0.4);
```

Comment:
cc @jonbell-lot23 in case this change is controversial. 

I wanted two distinct levels of "dimmed" for this UI, while the text also still remained somewhat legible.

Reply:
Interesting, thanks for flagging! Don’t have an opinion at this state, should be ok